src/templates/app/_includes/authenticate_message.jsx

```diff
@@ -156,6 +156,8 @@ const AuthenticateMessage = () => (
             ]}
         />
 
+        <SeparatorLine className='gr-padding-10' invisible />
```

Comment:
it should also have the class `data_show='mt5fin:vanuatu, labuan'`, because we want this padding only when the selfie section shows.